### PR TITLE
Contract call/create events

### LINKED ISSUE #56: Event Info in Contract Module
be able to show events for contract deploy and call transactions

https://github.com/aeternity/protocol/blob/master/contracts/sophia_explained.md#events

### PR DESCRIPTION
**Pull request by d-velev**
closes #56 
Contract call/create now also returns a list of logs(events) when successful

### CODE REVIEW

File: apps/core/test/core_contract_test.exs [hunk 1]
```diff
@@ -11,18 +11,22 @@ defmodule CoreContractTest do
           secret:
             "a7a695f999b1872acb13d5b63a830a8ee060ba688a478a08c6e65dfad8a01cd70bb4ed7927f97b51e1bcb5e1340d12335b2a2b12c8bc5221d63c4bcb39d41e61"
         },
-        "my_test",
```
Comment (thepiwo) [inline apps/core/test/core_contract_test.exs]: why not use the local node any more?

Comment (d-velev) [reply apps/core/test/core_contract_test.exs]: I've forgotten to revert it after testing, my bad

**Comment by thepiwo** [comment]
@DanielaIvanova @Artur64 can you review?

**Comment by d-velev** [comment]
It isn't present in any of the structure anymore, not sure if it's used somewhere in the node internally but we shouldn't care

**Pull request merged**